_modules/week-7.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Week 7 - State Relationship With Digital Commons
+---
+
+Slides: TBD
+
+Bibliography
+: **VIDEO**{: .label .label-purple }
+  : [Public Funds, Public Code](https://www.youtube.com/watch?v=iuVUzg6x2yo)
```

Comment:
Can you adapt the bibliography section to reflect the [actual bibliography](https://docs.google.com/document/d/1exXJnhWLa8VTamIMfb_uBwpK3RTVpnoQ1OYUsbd17f8/edit#heading=h.5qc2d7ais164)? Thanks